src/lib/envvar/input.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * @flow
+ * @format
+ */
+
+/**
+ * External dependencies
+ */
+import chalk from 'chalk';
+import { BooleanPrompt, prompt } from 'enquirer';
+
+/**
+ * Internal dependencies
+ */
+
+export function cancel() {
+	console.log( chalk.yellow( 'Command cancelled by user.' ) );
+	process.exit();
+}
+
+export async function confirmOrCancel( message: string ): Promise<void> {
+	const confirmed = await new BooleanPrompt( { message } )
+		.run()
+		.catch( cancel );
+
+	if ( ! confirmed ) {
+		cancel();
+	}
+}
+
+export function printBox( lines: string[] ): void {
```

Comment:
This is great!

I'm far from being a terminal CLI output expert but I wonder what a single line option might look like:
```
vvvvvvv----------- Received environment variable file contents -----------vvvvvvv 
line 1
line 2
line 3
^^^^^^^-----------             End of file contents            -----------^^^^^^^
```

Reply:
One other comment is it might be helpful to add in the telemetry (`trackEvent`) like we have in the other commands.

Reply:
1. I simplified this to `=====` markers.
2. Telemetry added.